spec/factories/taxi_requests.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :taxi_request do
+    passenger_id { Faker::Number.number }
+    address { Faker::Lorem.sentence[0..100] }
```

Comment:
```suggestion
    address { Faker::Lorem.sentence.first(100) }
```
Ruby와 Rails를 쓰다보면 배열 접근자를 거의 안쓰게 됩니다. 
되도록이면 해당 기능을 구현해놓은 method를 쓰는게 좋을 것 같습니다.